node/src/reactor/main_reactor/validate.rs

```diff
@@ -35,7 +35,13 @@ impl MainReactor {
                 self.control_logic_default_delay.into(),
             );
         }
-        if self.switch_block_header.is_none() {
+        if false
+            == self
+                .storage
+                .read_highest_block_header()
+                .unwrap_or(None)
+                .map_or(false, |header| header.is_switch_block())
```

Comment:
Probably a `ValidateInstruction::Fatal` is appropriate here too.

Reply:
Done